src/mugs.js

```diff
@@ -387,7 +387,13 @@ define([
                 if (lang !== defaultLang && disp === defaultDisp) {
                     disp += " [" + defaultLang + "]";
                 }
-                return $('<div>').text(disp).html();
+                if (this.options.richtext) {
```

Comment:
There's not one at the moment, but I added richtext to the mugoptions as it seemed slightly more self documenting and I was at the time considering making it possible to test case data and labels separately.